ui/guest/src/react/Guest.tsx

```diff
@@ -420,7 +420,9 @@ function Guest(props: GuestProps) {
         .subscribe((e) => {
           e.preventDefault();
           e.stopPropagation();
-          dispatch(desktopAssetDragStarted({ asset: e.dataTransfer.items[0] }));
+          if (e.dataTransfer.items[0].type.startsWith('image') && e.dataTransfer.items[0].kind === 'file') {
+            dispatch(desktopAssetDragStarted({ asset: e.dataTransfer.items[0] }));
+          }
```

Comment:
Desktop asset **and** assets panel drag & drop continues to work with this change applied?

Reply:
this only affect drag for desktop, i tested on my machine and it is working, but we need to test on your laptop to see if it continue working as expected on mac